calico/getting-started/kubernetes/managed-public-cloud/aks.md

```diff
@@ -33,7 +33,7 @@ To enable {{site.prodname}} network policy enforcement, follow these step-by-ste
 The geeky details of what you get:
 {% include geek-details.html details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:Calico,Datastore:Kubernetes' %}
 
-1. Create an Azure AKS cluster with no Kubernetes CNI pre-installed. Please refer to [Bring your own CNI with AKS](https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli) for details.
+1. Create an Azure AKS cluster with no Kubernetes CNI pre-installed. Note that `Bring your own Container Network Interface (CNI) plugin` is a preview feature with AKS. Please refer to [Bring your own CNI with AKS](https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli) for details.
```

Comment:
I don't think this is the right place to say it - we should use an alert to notify users of this.

Reply:
e.g.,: 

> Note: The ability to bring your own CNI plugin is a preview feature in AKS.

Reply:
I thought it would be connecting with the doc link better if I just insert the `Note` right before it. But you are right maybe I should just use an alert.